tests/settings/test_settings.py

```diff
@@ -69,6 +81,8 @@ def test_reset_settings(empty_index):
     response = index.get_settings()
     for rule in DEFAULT_RANKING_RULES:
         assert rule in response['rankingRules']
+    for typo in DEFAUT_TYPO_TOLERANCE:
+        assert typo in response['typoTolerance']
```

Comment:
No, I forgot to check the value, I will add this check either, except if you see something more accurate:
```python
assert DEFAULT_TYPO_TOLERANCE.get(typo) == response['typoTolerance'][typo]
```

Reply:
Only a slight modification suggestion since you know the `DEFAULT_TYPO_TOLERANCE` will contain the key for sure.

```py
assert DEFAULT_TYPO_TOLERANCE[typo] == response['typoTolerance'][typo]
```